packages/cli/src/commands/build/buildBackend.ts

```diff
@@ -29,17 +29,19 @@ interface BuildBackendOptions {
   targetDir: string;
   skipBuildDependencies: boolean;
   configPaths?: string[];
+  minify?: boolean;
 }
 
 export async function buildBackend(options: BuildBackendOptions) {
-  const { targetDir, skipBuildDependencies, configPaths } = options;
+  const { targetDir, skipBuildDependencies, configPaths, minify } = options;
   const pkg = await fs.readJson(resolvePath(targetDir, 'package.json'));
 
   // We build the target package without generating type declarations.
   await buildPackage({
-    targetDir: options.targetDir,
+    targetDir,
     packageJson: pkg,
     outputs: new Set([Output.cjs]),
+    minify,
```

Comment:
This would only minify the backend package itself, leaving any other packages in the monorepo without menification. I think it's likely best to forward the minification option at least to any standard builds, but perhaps skip it for any non-standard custom build scripts. See https://github.com/backstage/backstage/blob/5c3e1865a8994c9b1c221a441bbddfbdc92e8d5a/packages/cli/src/lib/packager/createDistWorkspace.ts#L208

Reply:
Thanks, makes sense!

Unsure if I need to forward the option [here](https://github.com/backstage/backstage/blob/master/packages/cli/src/commands/buildWorkspace.ts#L29-L32) though.